tests/prepareBaseImage-false_test.js

```diff
@@ -0,0 +1,94 @@
+const DebugCatcher = require('../tools/debugCatcher');
+
+Feature('Global set: prepareBaseImage: false');
+
+Scenario(
+  'R16 config = false, options = true, baseImageExists = false ',
+  async ({ I }) => {
+    I.say('New base image will be created');
+    const debugCatcher = new DebugCatcher();
```

Comment:
Why are you using helper as standard class? Why is helper not instanced in config file by standard way?

Reply:
Discussed - this is ok this way.